scripts/flush.py

```diff
@@ -1,18 +1,19 @@
-import requests
 import time
 import sys
 import os
 
-sys.path.append(os.getcwd())
-
-from app import config
-from uuid import uuid4
-from app.eq import build_response_id
+import requests
 from sdc.crypto.encrypter import encrypt
-from app import jwt
+from uuid import uuid4
+
+sys.path.append(os.getcwd())  # NB: script needs to know about app so append to PYTHONPATH
```

Comment:
```suggestion
sys.path.append(os.path.join(os.path.dirname(__file__), os.path.pardir)) # NB: script needs to know about app so append to PYTHONPATH
```
Should achieve the same without depending on the cwd the script is run with?